src/cript/api/api.py

```diff
@@ -194,7 +191,11 @@ def _get_s3_client(self) -> boto3.client:  # type: ignore
 
         identity_id = auth.get_id(IdentityPoolId=self._IDENTITY_POOL_ID, Logins={self._COGNITO_LOGIN_PROVIDER: self._token})
 
-        aws_credentials = auth.get_credentials_for_identity(IdentityId=identity_id["IdentityId"], Logins={self._COGNITO_LOGIN_PROVIDER: self._token})
+        aws_token = self._token
+        if len(aws_token.split()) > 1:
+            # Remove ID prefix like 'Bearer'
+            aws_token = aws_token.split()[-1]
```

Comment:
I am ok, with this being temporary.

It just wasn't working at the moment, so it needed a fix.

I disagree, that `strip` would be better. I thought about it.

Strip only works, if it actually starts with "Bearer", but with how unstable the backend is (especially tokens), I don't want to rely on this. (It used to be a "Token" prefix, and the technology might change again.
(And good luck debugging this from users, since users won't share there token with you.)

The only assumption the split version makes, is that there are no spaces in the token.
And I think that is a very minimal assumption, we have to make.

It also works if there is no prefix, or anything.

You just need no spaces in the token, and the token is last (if there are more sections).

That is why I chose this method over `strip`.

Reply:
I don't know. This is just temporary anyways.
So we should just use something that works. and make it note to do it right, once we can.

Reply:
okay that makes sense